lib/Map/Vector/geoRssConvertor.js

```diff
@@ -157,6 +157,7 @@ function getGeometryFromWhere(node) {
       result = getGeometryFromWhere(child);
     }
   }
+  return result;
```

Comment:
@zoran995 this code is from 2020 in commit 8ebc61a6, do you remember if your intention was to return undefined when falling out of the loop?